src/Command/Docker/Build.php

```diff
@@ -170,100 +95,31 @@ protected function configure()
             )->addOption(
                 self::OPTION_SYNC_ENGINE,
                 null,
                 InputOption::VALUE_REQUIRED,
-                sprintf(
-                    'File sync engine. Works only with developer mode. Available: (%s)',
-                    implode(', ', DeveloperCompose::SYNC_ENGINES_LIST)
-                ),
-                DeveloperCompose::SYNC_ENGINE_DOCKER_SYNC
+                sprintf('File sync engine. Works only with developer mode.')
             );
 
         parent::configure();
     }
 
     /**
-     * {@inheritdoc}
+     * {@inheritDoc}
      *
-     * @throws ConfigurationMismatchException
-     * @throws FileSystemException
-     * @throws UndefinedPackageException
-     * @throws ServiceMismatchException
+     * @throws ShellException
      */
     public function execute(InputInterface $input, OutputInterface $output)
     {
-        $type = $input->getOption(self::OPTION_MODE);
-        $syncEngine = $input->getOption(self::OPTION_SYNC_ENGINE);
-
-        $compose = $this->composeFactory->create($type);
-        $config = $this->configFactory->create();
-
-        if (ComposeFactory::COMPOSE_DEVELOPER === $type
-            && !in_array($syncEngine, DeveloperCompose::SYNC_ENGINES_LIST, true)
-        ) {
-            throw new ConfigurationMismatchException(sprintf(
-                "File sync engine '%s' is not supported. Available: %s",
-                $syncEngine,
-                implode(', ', DeveloperCompose::SYNC_ENGINES_LIST)
-            ));
-        }
-
-        array_walk(self::$optionsMap, static function ($key, $option) use ($config, $input) {
-            if ($value = $input->getOption($option)) {
-                $config->set($key, $value);
-            }
-        });
-
-        $config->set(DeveloperCompose::SYNC_ENGINE, $syncEngine);
-
-        if (in_array(
-            $input->getOption(self::OPTION_MODE),
-            [ComposeFactory::COMPOSE_DEVELOPER, ComposeFactory::COMPOSE_PRODUCTION],
-            false
-        )) {
-            $versionList = $this->serviceConfig->getAllServiceVersions($config);
-            $errorList = $this->validator->validateVersions($versionList);
+        $output->writeln('<error>This command is deprecated.</error>');
 
-            $helper = $this->getHelper('question');
-            $question = new ConfirmationQuestion(
-                'There are some service versions which are not supported'
-                . ' by current Magento version:' . "\n" . implode("\n", $errorList) . "\n"
-                . 'Do you want to continue?[y/N]',
-                false
-            );
-
-            if ($errorList && !$helper->ask($input, $output, $question) && $input->isInteractive()) {
-                return 1;
-            }
-
-            $this->distGenerator->generate();
-        }
+        $command = str_replace(self::NAME, 'build:compose', (string)$input);
+        $process = $this->shell->execute('./vendor/bin/ece-docker ' . $command);
```

Comment:
Wouldn't it be easier & cleaner to just call `Magento\CloudDocker\Command\BuildCompose::run($input, $output)`? Plus there'd be no casting `$input` to a string and Phpstan wouldn't complain

Reply:
No, because it's a different package with different entry points

Reply:
How does that make it any different than calling a method from any other package?

Reply:
Dependency Injection instantiation need to come along to call those methods

Reply:
Got it, you're right. Thanks!